### PR TITLE
kernelci.cli: rewrite split_attributes() method

### PR DESCRIPTION
**Pull request by gctucker**
Add a new kernelci.cli.split_attributes() method to be used with the
    Click command line tools.  This is based on the previous
    implementation with symbolic comparison operators but with a revised
    implementation.  Update unit tests accordingly.

### CODE REVIEW

**Comment by gctucker** [comment]
@nuclearcat Would you like to review this one since you introduced the feature of boolean operator symbols?

**Comment by nuclearcat** [comment]
Absolutely!

**Review by nuclearcat — APPROVED** [review]
LGTM

File: kernelci/cli/__init__.py [hunk 1]
```diff
@@ -129,3 +132,48 @@ def group(self, *args, cls=None, **kwargs):
 def kci(ctx, settings):
     """Entry point for the kci command line tool"""
     ctx.obj = CommandSettings(settings)
+
+
+def split_attributes(attributes: typing.List[str]):
+    """Split attributes into a dictionary.
+
+    Split the attributes string into a dictionary using space as a delimiter
```
Comment (gctucker) [inline kernelci/cli/__init__.py:140]: Actually, I just realised there was a problem with the docstring.  I'll send a fixup.  The attributes argument is not a string with spaces but a list of strings already split.

**Pull request merged**